app/src/main/java/com/internship/tvseries/data/repository/config/ConfigBackendRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.internship.tvseries.data.repository.config;
+
+import com.internship.tvseries.data.model.Configuration;
+
+import java.util.function.Consumer;
+
+public class ConfigBackendRepository implements ConfigRepository {
```

Comment:
I know that this is just a temporary name but I guess we can rename it LynxConfigRepository